dts/msm8916/msm8916-huawei-hwt1a21l.dts

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "skeleton.dtsi"
+
+/ {
+	qcom,msm-id = <206 0>;
+	qcom,board-id = <8065 0> , <8065 4>;
```

Comment:
Please include only the board-id you have, probably:

```suggestion
	qcom,board-id = <8065 4>;
```

Also, can you post the `lk_log` or UART (see README) so I can check if everything looks okay?